src/jdk.management.agent/share/classes/jdk/internal/agent/spi/AgentProvider.java

```diff
@@ -33,27 +33,13 @@
 
     /**
      * Instantiates a new AgentProvider.
```

Comment:
This class should probably be removed altogether. I see that you logged [JDK-8283254](https://bugs.openjdk.java.net/browse/JDK-8283254) so this is fine.

Reply:
Thanks Daniel - yes will get rid of this class soon, I'll keep these two actions in separate changes. 8-)